src/Sulu/Bundle/AdminBundle/Resources/js/components/Login/Login.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import React from 'react';
+import Icon from '../../components/Icon';
+import {translate} from '../../utils';
+import LoginForm from './LoginForm';
+import loginStyles from './login.scss';
+import logo from './logo.svg';
+
+const BACK_LINK_ARROW_LEFT_ICON = 'chevron-left';
+
+type Props = {
+    onLogin: (user: string, password: string) => void,
+    onResetPassword: (user: string) => void,
+};
+
+export default class Login extends React.PureComponent<Props> {
+    handleLogin = (user: string, password: string) => {
```

Comment:
Do we need that method? It just adds overhead, doesn't it? We could simply pass the `onLogin` part to the `LoginForm` (or does the eslint rule complain then? I thought in that case it was working...)